src/main/java/frc/robot/stateManagement/NoteState.java

```diff
@@ -0,0 +1,28 @@
+package frc.robot.stateManagement;
+
+import java.util.Arrays;
+
+public enum NoteState {
+  NO_TE(0),
+  LOADING(10),
+  CHAMBER(20),
+  READY_TO_FIRE(30),
+  AMP(40);
```

Comment:
What does AMP mean here?  Ready for amp placement?  Might want to follow the READY_TO_FIRE pattern, and put READY_TO_ in front of amp.

Reply:
Talking to Malachi - READY_TO_PLACE?